WordPress/gutenberg
# Pagination Block: Fix inconsistent margins between editor and frontend

Fixes https://github.com/woocommerce/woocommerce/issues/42204
Fixes https://github.com/WordPress/gutenberg/issues/51299

## What?

* Remove default margins from pagination previous, next and number elements.
* Add spacing controls

## Why?

Fix the inconsistent margins between the editor and the front-end.

## How?

By removing offending CSS rules.

## Testing Instructions
1. Add Query Loop to a new post or page.
2. In your block settings set a custom query to reduce the posts per page if needed to ensure the pagination appears on the frontend.
3. Set a background color to the pagination block so you can see spacing clearly.
4. Ensure the spacing is now consistent between the frontend and the editor.

| State | Editor | Frontend |
|--------|--------|--------|
| After | ![Screenshot 2024-08-28 at 16 16 38](https://github.com/user-attachments/assets/f5e34a65-0ad7-4ae8-addc-cd73fc2f60ea) | ![Screenshot 2024-08-28 at 16 16 28](https://github.com/user-attachments/assets/14e2f6bc-c3ee-4c92-88e1-520b6b8c8e56) |
| Before | ![Screenshot 2024-08-28 at 16 19 37](https://github.com/user-attachments/assets/af01d7f0-f06f-4086-855a-270d07cfbe53) | ![Screenshot 2024-08-28 at 16 19 49](https://github.com/user-attachments/assets/ff1b2de2-e94b-4e1f-b398-fa42b7763f6b) |

### Testing Instructions for Keyboard



## Screenshots or screencast
